Fix Vulkan-Site repository reference and filter build errors

gpx1000 opened the pull request:
Updated the workflow to reference the KhronosGroup/Vulkan-Site repository. Also improved the build script to filter out known missing document errors, preventing unnecessary build failures. This ensures a smoother build process and correct error reporting.

## Description

Please include a summary of the change, new sample or fixed issue. Please also include relevant motivation and context.
Please read the [contribution guidelines](https://github.com/KhronosGroup/Vulkan-Samples/tree/main/CONTRIBUTING.adoc)

Fixes #1219 

## General Checklist:

Please ensure the following points are checked:

- [ ] My code follows the [coding style](https://github.com/KhronosGroup/Vulkan-Samples/tree/main/CONTRIBUTING.adoc#Code-Style)
- [ ] I have reviewed file [licenses](https://github.com/KhronosGroup/Vulkan-Samples/tree/main/CONTRIBUTING.adoc#Copyright-Notice-and-License-Template)
- [ ] I have commented any added functions (in line with Doxygen)
- [ ] I have commented any code that could be hard to understand
- [ ] My changes do not add any new compiler warnings
- [ ] My changes do not add any new validation layer errors or warnings
- [ ] I have used existing framework/helper functions where possible
- [ ] My changes do not add any regressions
- [ ] I have tested every sample to ensure everything runs correctly
- [ ] This PR describes the scope and expected impact of the changes I am making

 Note: The Samples CI runs a number of checks including:
 - [ ] I have updated the header Copyright to reflect the current year (CI build will fail if Copyright is out of date)
 - [ ] My changes build on Windows, Linux, macOS and Android. Otherwise I have [documented any exceptions](https://github.com/KhronosGroup/Vulkan-Samples/tree/main/CONTRIBUTING.adoc#General-Requirements)

 If this PR contains framework changes:
 - [ ] I did a full batch run using the `batch` command line argument to make sure all samples still work properly

## Sample Checklist

If your PR contains a new or modified sample, these further checks must be carried out *in addition* to the General Checklist:
- [ ] I have tested the sample on at least one compliant Vulkan implementation
- [ ] If the sample is vendor-specific, I have [tagged it appropriately](https://github.com/KhronosGroup/Vulkan-Samples/tree/main/CONTRIBUTING.adoc#General-Requirements)
- [ ] I have stated on what implementation the sample has been tested so that others can test on different implementations and platforms
- [ ] Any dependent assets have been merged and published in downstream modules
- [ ] For new samples, I have added a paragraph with a summary to the appropriate chapter in the readme of the folder that the sample belongs to [e.g. api samples readme](https://github.com/KhronosGroup/Vulkan-Samples/blob/main/samples/api/README.adoc)
- [ ] For new samples, I have added a tutorial README.md file to guide users through what they need to know to implement code using this feature. For example, see [conditional_rendering](https://github.com/KhronosGroup/Vulkan-Samples/tree/main/samples/extensions/conditional_rendering)
- [ ] For new samples, I have added a link to the [Antora navigation](https://github.com/KhronosGroup/Vulkan-Samples/blob/main/antora/modules/ROOT/nav.adoc) so that the sample will be listed at the Vulkan documentation site

gpx1000 committed:
Fix Vulkan-Site repository reference and filter build errors

Updated the workflow to reference the KhronosGroup/Vulkan-Site repository. Also improved the build script to filter out known missing document errors, preventing unnecessary build failures. This ensures a smoother build process and correct error reporting.

gpx1000 committed:
revert this part of the change for now.  Will need to investigate what needs to happen to allow this to not point to my repo.

SaschaWillems commented on .github/workflows/build-Antora-doc.yml:
```diff
@@ -80,11 +80,25 @@ jobs:
+          output = "$(npx antora antora-parent-ci-playbook.yml --stacktrace | \
+            jq 'select(.msg | contains(" not found: guide:ROOT:index.adoc") | not)' | \
+            jq 'select(.msg | contains(" not found: tutorial:ROOT:00_Introduction.adoc") | not)' | \
+            jq 'select(.msg | contains(" not found: https://github.com/wolfpld/tracy/releases/tag/v0.10") | not)' | \
```
The error for tracy was correct ;) I fixed those links in #1221.

gpx1000 replied:
Nice, we're gonna have to get yours accepted first and then I'll grab it here.

gpx1000 committed:
Fix syntax error in Antora build script

Correct missing space around the assignment operator in the Antora build workflow. This ensures the build output is correctly captured and processed.

gpx1000 committed:
Assume that #1221 will be accepted before this one.

gpx1000 committed:
enforce the log-format of JSON.

gpx1000 committed:
Improve error filtering in Antora build workflow

Replaced 'jq' with 'grep' and 'jq -R' for better JSON handling. Updated filter conditions to exclude specific error messages from the Antora build output.

gpx1000 committed:
Update build workflow to filter out JSON output

This change adds a `-r` option to the `jq` command in the Antora build workflow. The `-r` flag ensures the JSON output is properly processed, improving clarity and function of JSON filtering.

gpx1000 committed:
Update Antora build log level to error and format output

Changed Antora build command to use `--log-level=error` for cleaner logs. Enhanced JSON output handling with `pino-pretty` for more readable error messages. This should help in better identifying and troubleshooting errors in the build process.

gpx1000 committed:
Refine logging in build-Antora-doc workflow

Removed unnecessary JSON formatting step with pino-pretty in the Antora documentation build workflow. This change simplifies the log output by directly parsing it with jq.

gpx1000 committed:
Enhance log readability with pino-pretty

Replace `jq -r .` with `jq -r | npx pino-pretty` to format JSON logs for better readability. This change helps in diagnosing issues by making log outputs more user-friendly and easier to interpret.

gpx1000 committed:
Merge branch 'KhronosGroup:main' into antora-doc-fix

gpx1000 committed:
Remove unnecessary pino-pretty from JSON processing

The pino-pretty step was removed from the JSON processing pipeline in the GitHub workflow `build-Antora-doc.yml`. This change simplifies the workflow and ensures that the JSON output is handled as raw data.

gpx1000 committed:
Merge remote-tracking branch 'origin/antora-doc-fix' into antora-doc-fix

gpx1000 committed:
Fix syntax error in Jenkins JSON processing

Corrected the JSON processing command in the Jenkins build workflow by adding a period after jq -r. This ensures the command runs without syntax errors and filters log messages as intended.

gpx1000 committed:
Simplify Antora log filtering in build workflow

Removed redundant flag '-r' from the jq command in the Antora log filtering process. This change streamlines the command sequence, making it easier to read and maintain without altering its functionality.

gpx1000 committed:
try it without the raw input?

gpx1000 committed:
Update jq syntax for better error filtering

Modified the jq syntax in the Antora build workflow to correct the usage of string contains function. This ensures more accurate filtering of specific document not found errors while running the Antora build process.

gpx1000 committed:
Suppress grep errors in build-Antora-doc.yml

Redirect stderr to /dev/null in the grep command to suppress error messages. This ensures cleaner log outputs by preventing grep-related errors from being displayed.

gpx1000 committed:
Fix syntax error in jq filter statements

Corrected the jq filter syntax to properly handle and parse Antora log messages, ensuring the message keys are correctly referenced within the select function. This prevents missing guide and tutorial documents from being incorrectly reported.

gpx1000 committed:
try raw output

gpx1000 committed:
Refine Antora log processing in CI workflow

Removed irrelevant fields from Antora log output and streamlined error filtering. This enhances the clarity of log messages, making it easier to diagnose issues.

gpx1000 committed:
opps

gpx1000 committed:
Improve Antora workflow log filtering

Refactor the jq command to enhance readability and maintainability. The changes ensure all log entries containing specific 'not found' messages are excluded efficiently.

gpx1000 committed:
Fix missing closing parenthesis in jq command

The jq command was missing a closing parenthesis leading to potential syntax errors. This fix ensures proper filtering of error messages related to missing documentation files.
